packages/main/src/i18n/messagebundle.properties

```diff
@@ -208,6 +208,9 @@ MESSAGE_STRIP_INFORMATION=Information Bar
 #XFLD: MultiComboBox dialog button
 MULTICOMBOBOX_DIALOG_OK_BUTTON=OK
 
+#XMSG: Text used for reporting that a radio group requires one of the radio buttons to be checked
```

Comment:
I suggest :
#XMSG: Text used for reporting that a radio button group requires one of the radio buttons to be checked